fixposition_odometry_converter_ros2/launch/odom_converter.yaml

```diff
@@ -1,7 +1,7 @@
 /**:
   ros__parameters:
-    topic_type: "Odometry" # Twist, TwistWithCov or Odometry
+    topic_type: "Twist" # Twist, TwistWithCov or Odometry
     input_topic: "/robot/odom" # Original topic name
     fixposition_speed_topic: "/fixposition/speed" # Fixposition topic name, to be subscribed by the fixposition_driver
     multiplicative_factor: 1000 # To convert from the original unit of measurement to mm/s
-    use_angular: false # If the angular velocity of the speed message is filled
+    use_dimensions: 1 # 1 to use x only, 2 to use x and y, 3 to use x y z
```

Comment:
I don't see the need for any of this configurability in this node, except for the input topic. 

Lower configurability = fewer customer mistakes 

- type can be auto-deduced
- fixposition_speed_topic is constant
- Multiplicative factor is not needed (can be constant to 1000)
- use_dimensions should ideally be `x, y, z` if we're converting from standard ros messages